godot-kotlin/godot-library/src/nativeGen/kotlin/godot/Resource.kt

```diff
@@ -1,5 +0,0 @@
-package godot
-
-//TODO: this needs to be properly implemented. It's just here for the implementation of the PropertyTypeHintAnnotations
-open class Resource {
```

Comment:
Because this was deleted, the sample cannot build atm. Either we restore this file or remove the corresponding testing lines in the test class inside the samples

Reply:
Oh yes, that's seems to be a git error I did when switching from api generation branch ^^'